tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -21,6 +22,8 @@ def option_price(volatilities,
                  expiries,
                  spots=None,
                  forwards=None,
+                 risk_free_rates=None,
```

Comment:
We have some other contribution pending and would like to keep naming consistent. Please rename `risk_free_rates` to `discount_rates`